mint/mock_test.go

```diff
@@ -3,10 +3,9 @@ package mint
 import (
 	"context"
 	"crypto/sha256"
-	"math/rand"
-
 	"github.com/lightninglabs/aperture/lsat"
 	"github.com/lightningnetwork/lnd/lntypes"
+	"math/rand"
```

Comment:
same comment re imports: `math/rand` belongs in the standard lib section and new line should stay